app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/common/SurveyText.kt

```diff
@@ -11,19 +11,20 @@ import androidx.compose.ui.unit.TextUnit
 import com.kks.nimblesurveyjetpackcompose.ui.theme.NeuzeitFamily
 
 @Composable
-fun SurveyBoldText(
+fun SurveyText(
```

Comment:
Seems like this change should have been mentioned in the PR description, it seems pretty major since many files are affected 🤔

Reply:
Updated